.chloggen/jm-otel-prom-metadata-translation-pass-namespace-config.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
I would call this `bug_fix`, since the existing behavior of correlating metric names and metric metadata is broken 🤔

Reply:
done